docs/conf.py

```diff
@@ -35,6 +35,10 @@
 # The encoding of source files.
 #source_encoding = 'utf-8-sig'
 
+# Disable the use of smartquotes, which converts quotes and dashes to
+# typographically correct entities.
```

Comment:
Maybe don't explain the pros of enabling it, but rather why it should be disabled.